en/docs/references/ballerina-expression-syntax.md

```diff
@@ -0,0 +1,421 @@
+# Ballerina Expression Syntax
+
+Ballerina expressions comprise a powerful expression syntax based on the [Ballerina language](https://ballerina.io/) for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you with the common expressions that you can use while developing Choreo applications.
+
+## Supported Literals 
+
+### String
+
+A string is simply a sequence of characters. It is required to use double quotes (`"`)  to mark the boundary between the starting and ending characters.
+
+#### Examples
+
+  ```ballerina
+  "Colombo"
+  ```
+  ```ballerina
+  "12345"
+  ```
+  ```ballerina
+  "Ballerina is a programming language intended for network-distributed applications."
+  ```
+
+### Signed Integer
+
+A signed integer is an integer with the  `+` or `-` character in the front, indicating the sign. You can also define a positive integer or zero using only numerical characters without the sign. Using the signed integer type, you can give positive integers, negative integers, and zero as an input to an expression-supported field.
+
+#### Examples
+
+  ```ballerina
+  1234
+  ```
+  ```ballerina
+  -10
+  ```
+  ```ballerina
+  +2020
+  ```
+
+### Decimal Floating-Point Number
+
+A decimal floating-point number is a number with a decimal point. You have the option to use `+` or `-` characters in the front to indicate the sign.
+
+#### Examples
+
+  ```ballerina
+  0.3353
+  ```
+  ```ballerina
+  2044.24
+  ```
+  ```ballerina
+  -8593.992
+  ```
+  ```ballerina
+  +950.930
+  ```
+  ```ballerina
+  0
+  ```
+
+### Boolean Value
```

Comment:
Same as above. Do we need to say `Value` here?

Reply:
yeah we can remove that.